tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -1141,6 +1141,79 @@ public function test_sync_jetpack_publish_post_works_with_interjecting_plugins()
 		$this->assertEquals( $events[3]->action, 'jetpack_published_post' );
 	}
 
+	public function test_sync_feedback_fields_when_feedback_submitted() {
+		/*
+		 * Note that a test for ensuring the proper meta is set when the feedback
+		 * form is submitted was added here,
+		 *
+		 * tests/php/modules/contact-form/test_class.grunion-contact-form.php
+		 * test_process_submission_will_store_all_field_metadata()
+		 *
+		 * In the test below, we will just set the meta and ensure it is
+		 * synced, rather than duplicate the feedback form test
+		 */
+
+		$user_id = $this->factory->user->create();
+		$post_id = $this->factory->post->create( array( 'post_author' => $user_id, 'post_type' => 'feedback' ) );
+
+		$feedback_fields = array(
+			'g4-name' => array
+			(
+				'key'     => 'g4-name',
+				'value'   => 'John Doe',
+				'type'    => 'name',
+				'label'   => 'Name',
+				'options' => array(),
+				'post_id' => $post_id
```

Comment:
Do we need to store the $post_id here? 
I am not sure how that post ID is different from the feedback post Id that gets created?

Reply:
See answer to same question below, https://github.com/Automattic/jetpack/pull/10185#pullrequestreview-159087313